docs/guides/hooks-list.rst

```diff
@@ -410,7 +410,37 @@ Other
 	``entity_subtype`` will be passed with the ``$params`` array to the callback.
 
 **entity:<icon_type>:sizes, <entity_type>**
-	Allows filtering sizes for custom icon types, see ``entity:icon:sizes, <entity_type>``
+	Allows filtering sizes for custom icon types, see ``entity:icon:sizes, <entity_type>``.
+
+	The hook must return an associative array where keys are the names of the icon sizes
+	(e.g. "large"), and the values are arrays with the following keys:
+	 * ``w`` - Width of the image in pixels
```

Comment:
How did you catch this? I didn't get any build errors locally, and the generated HTML seemed valid.